site/app/views/submission/HomeworkView.php

```diff
@@ -197,6 +197,22 @@ public function renderLateDayMessage(LateDays $late_days, Gradeable $gradeable,
             $active_days_charged = max(0, $active_days_late - $extensions);
         }
 
+
+        $DayLightSavingMessageRequired = false;
+        $date = new \DateTime();
+        $daylightSpring = new \DateTime('2024-05-17', $this->core->getConfig()->getTimezone());
+        $daylightFall = new \DateTime('2024-11-03', $this->core->getConfig()->getTimezone());
```

Comment:
Fix the variables names. The project's [PHP style guide](https://submitty.org/developer/software_and_system_design/coding_style_guide/php#naming-conventions) indicates variables should use snake case.

Reply:
Fixed I believe!